openedx/core/djangoapps/course_live/providers.py

```diff
@@ -92,8 +92,42 @@ class Zoom(LiveProvider):
     ]
 
     @property
-    def is_enabled(self):
+    def has_free_tier(self) -> bool:
+        """
+        Check if free tier is enabled by checking for valid keys
+        """
+        return self.has_valid_global_keys()
+    
+    @property
+    def is_enabled(self) -> bool:
         return True
+    
+    @staticmethod
+    def get_global_keys() -> Dict:
+        """
+        Get keys from settings
+        """
+        try:
+            COURSE_LIVE_GLOBAL_CREDENTIALS = {
```

Comment:
If values are not set, it'll throw an error. I suggest initializing these values in common.py and production.py. It's also easier for developers to check which environmental variables are required to run this app.